src/test/java/ArrayQueueTest.java

```diff
@@ -47,27 +47,31 @@ void enqueuingOneItemIsFirst() {
     }
 
     @Test
-    @DisplayName("Enqueuing 6 elements into the queue expands capacity.")
-    void enqueuingBeyondCapacityCallsExpandCapacityToMakeRoom() {
-        Queue<Integer> queue = new ArrayQueue<>(5);
+    @DisplayName("Enqueueing and Dequeuing returns in FIFO order.")
+    void enqueueingAndDequeuingReturnsElementsInFIFOOrder() {
+        Queue<Integer> queue = new ArrayQueue<>();
         for (int i = 0; i < 6; ++i) {
             queue.enqueue(i);
         }
-        for (int i = 0; i < 5; ++i) {
-            queue.dequeue();
+        for (int i = 0; i < 6; ++i) {
+            assertEquals(i, queue.dequeue());
         }
-        // After 6 enqueues, expandCapacity is called
-        // After 5 dequeues, the 6th element will be first
-        assertEquals(5, queue.first());
     }
 
     @Test
-    @DisplayName("Enqueueing and Dequeuing returns in FIFO order.")
-    void enqueueingAndDequeuingReturnsElementsInFIFOOrder() {
-        Queue<Integer> queue = new ArrayQueue<>();
+    @DisplayName("Enqueuing 6 elements into the queue expands capacity.")
+    void enqueuingBeyondCapacityCallsExpandCapacityToMakeRoom() {
```

Comment:
Names, I'd include something like "while maintaining queue order" so it's clear that it's not just a simple capacity check.